lib/src/formatters/frameworks/json/vue-i18n.ts

```diff
@@ -0,0 +1,135 @@
+import JavascriptOutputFile from "../../shared/fileTypes/JavascriptOutputFile";
+import OutputFile from "../../shared/fileTypes/OutputFile";
+import { applyMixins } from "../../shared";
+import javascriptCodegenMixin from "../../mixins/javascriptCodegenMixin";
+import JSONOutputFile from "../../shared/fileTypes/JSONOutputFile";
+import BaseFramework from "./base";
+
+export default class VueI18nFramework extends applyMixins(
+  BaseFramework,
+  javascriptCodegenMixin
+) {
+  process(
+    outputJsonFiles: Record<string, JSONOutputFile<{ variantId: string }>>
+  ) {
+    // vue-18n requires single mustaching for their json inputs, so we need to update the json
+    for (const file of Object.values(outputJsonFiles)) {
+      for (const key in file.content) {
+        const content = file.content[key];
+        if (typeof content === "string") {
+          file.content[key] = content
+            .replaceAll(/{{/g, "{")
+            .replaceAll(/}}/g, "}");
+        }
+      }
+    }
+
```

Comment:
Am I correct in assuming that this is the only difference between the `i18next` formatter and the `vue-i18n` formatter?

Reply:
yup!